Caboodle/DataTransfer/DataTransfer.uwp.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Foundation;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class DataTransfer
+    {
+        static ShareTextRequest shareRequest;
```

Comment:
Is this safe? Maybe rather than using a static field, just use an inline delegate?